### DISCUSSION TITLE
unable to add custom field

### DISCUSSION THREAD

**Discussion by nikunj-tele** [netbox-community/netbox#15167] [Help Wanted!] [answered]
Hi,
while adding custom field i am getting issue with object type as attached here.
tried to look device type but could not search appropriate field which could cause this issue.

appreicate your help!

![image](https://github.com/netbox-community/netbox/assets/55275995/af58ece1-bd73-46f1-8118-cd045291e1e1)

**Comment by PaulR282** [comment, marked as answer]
If you want to get a Boolean tag, you leave the Object type empty, as it's shown there. This is only to set a type for Object/Multi-Object fields.

**Reply by nikunj-tele** [reply]
Thanks for prompt response. 
it works :)
